docs/src/pages/demos/grid-list/tslint.json

```diff
@@ -0,0 +1,6 @@
+{
+  "extends": "../../../tslint.json",
```

Comment:
Great job so far. We're almost there: Do we need the explicit `extends` here? eslint resolves them hierarchically which means there could be a eslintrc one level above. If we would add a tslint.json one level above I would expect that it is ignored with this explicit `extends`. I would prefer the same behavior as eslint (considering tslint will become deprecated soon).

You can verify that it still `extends` without the explicit `extends` key by adding  `const foo: number = "1"; // $ExpectError` to a TS demo from this PR. If it works `yarn docs:typescript:check` should pass. If not it fails. When it fails we will have to keep the `extends` key.

Reply:
Without the `extends` `docs:typescript:check` fails with the following error:

> Tried to lint E:/Workspace/material-ui/docs/src/pages/demos/grid-list/AdvancedGridList.tsx but found no valid, enabled rules for this file type and file path in the resolved configuration.

Reply:
Too bad. Will need to be extra careful if we need to add more tslint.json. Thanks for trying.